File: Loiane_Training_C#/Basico/D/Ex14.cs
using System;

class Ex14
{
    static void Main(string[] args)
    {
        int quantidadePares = 0;
        for (int i = 0; i < 10; i++)
        {
            Console.WriteLine($"Digite {i + 1} o numero ");
            int number = int.Parse(Console.ReadLine());
            if (number % 2 == 0)
            {
                quantidadePares++;
            }
        }

        Console.WriteLine($"Pares: {quantidadePares} | Impares: {10 - quantidadePares}");
    }
}

File: loiane-training-java/basico/Exer Aula 16_17/Ex14.java
import java.util.Scanner;

public class Ex14 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        var number = 0;
        var quantityOfPair = 0;
        var quantityOfOdd = 0;
        for (int i = 0; i < 10; i++) {
            number = scan.nextInt();
            if (number % 2 == 0) {
                quantityOfPair++;
            } else {
                quantityOfOdd++;
            }
        }
        System.out.println("Pares: " + quantityOfPair + " | Impares: " + quantityOfOdd);
        scan.close();
    }
}
